b2sdk/utils/__init__.py

```diff
@@ -84,7 +89,7 @@ def choose_part_ranges(content_length, minimum_part_size):
     return parts
 
 
-def hex_sha1_of_stream(input_stream, content_length):
+def hex_sha1_of_stream(input_stream, content_length) -> Sha1HexDigest:
```

Comment:
here it's updated nicely, but in docstring it's still `rtype: str` - that should be removed or there's a conflict